pkg/providers/amifamily/ami.go

```diff
@@ -160,8 +160,28 @@ func (p *Provider) getDefaultAMIFromSSM(ctx context.Context, nodeTemplate *v1alp
 		if err != nil {
 			return nil, err
 		}
-		amis = append(amis, AMI{Name: ssmOutput.Name, AmiID: amiID, Requirements: ssmOutput.Requirements})
+		amis = append(amis, AMI{AmiID: amiID, Requirements: ssmOutput.Requirements})
 	}
+
+	ids := lo.Reduce(amis, func(agg string, item AMI, _ int) string {
```

Comment:
Can you comment what this is doing and why you are doing it. I understand but it may not be immediately clear to a reader